plugins/plugin-svelte/plugin.js

```diff
@@ -53,12 +53,16 @@ module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
       const {sourceMaps} = snowpackConfig.buildOptions;
       const output = {
         '.js': {
+          contents: js.code,
```

Comment:
re: larger comment above: walk back all changes to the plugins for now, to guarantee backward compat and simplify review